manageiq-providers-hawkular.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "hawkular-client", "~> 3.0.2"
+  s.add_runtime_dependency "hawkular-client", "~> 4.0.0"
```

Comment:
Yes we will retain the existing metrics functionality for now.
`~> 4.0.0` and `~> 3.0.2` cannot both be satisfied